internal/cli/commands/get/source_types.go

```diff
@@ -31,7 +31,7 @@ func SourceTypesCommand(bindplane *cli.BindPlane) *cobra.Command {
 		Short:   "Displays the source types",
 		Long:    `A source type is a type of source that collects logs, metrics, and traces.`,
 		RunE: getImpl(bindplane, "source-type", getter[*model.SourceType]{
```

Comment:
```suggestion
		RunE: getImpl(bindplane, "source-types", getter[*model.SourceType]{
```
This was already here, but we should be consistent with pluralized `resourceName`